<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TreePainter</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>560</width>
     <height>560</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>500</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>180</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid red;</string>
   </property>
   <property name="plainText">
    <string>a b
a c
b d
b e
c f</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>560</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>15</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>500</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>节点大小</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>530</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>垂直间距</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>560</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>水平间距</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
